prow/plugins/trigger/generic-comment_test.go

```diff
@@ -516,7 +557,43 @@ func TestHandleGenericComment(t *testing.T) {
 					},
 				},
 			},
-			ShouldReport: false,
+			ShouldReport: true,
+		},
+		{
+			name:   "branch-sharded job. no shard matches base branch. Skipped statuses elided.",
+			Author: "trusted-member",
+			Branch: "branch",
+			Body:   "/test jab",
+			State:  "open",
+			IsPR:   true,
+			Presubmits: map[string][]config.Presubmit{
+				"org/repo": {
+					{
+						JobBase: config.JobBase{
+							Name: "jab",
+						},
+						Brancher: config.Brancher{Branches: []string{"master"}},
+						Reporter: config.Reporter{
+							Context: "pull-jab",
+						},
+						Trigger:      `(?m)^/test (?:.*? )?jab(?: .*?)?$`,
+						RerunCommand: `/test jab`,
+					},
+					{
+						JobBase: config.JobBase{
+							Name: "jab",
+						},
+						Brancher: config.Brancher{Branches: []string{"release"}},
+						Reporter: config.Reporter{
+							Context: "pull-jab",
+						},
+						Trigger:      `(?m)^/test (?:.*? )?jab(?: .*?)?$`,
+						RerunCommand: `/test jab`,
+					},
+				},
+			},
+			ElideSkippedContexts: true,
```

Comment:
Can we have one test in this set that sets this to false and validate that a skipped job reports?

Reply:
It's false by default so actually for every single case where we could post a skipped context we have a test case in both directions